lib/plaid/user.rb

```diff
@@ -112,12 +166,29 @@ def transactions(pending: false, account_id: nil, start_date: nil, end_date: nil
     #
     # Returns self.
     def update(username, password, pin = nil)
```

Comment:
cool, the most important thing in my mind was that if we did use an `ActiveRecord`-like design, that we follow the same conventions around the methods we define. since we're doing that I feel more comfortable here.